res/css/views/rooms/_EditMessageComposer.pcss

```diff
@@ -33,7 +33,7 @@ limitations under the License.
         padding: 3px 6px;
 
         &:focus {
-            border-color: rgba($accent, 0.5); /* Only ever used here */
+            border-color: $accent-500;
```

Comment:
This will break custom themes e.g. https://github.com/aaronraimist/element-themes